frappe/desk/doctype/system_console/system_console.py

```diff
@@ -33,4 +33,9 @@ def run(self):
 def execute_code(doc):
 	console = frappe.get_doc(json.loads(doc))
 	console.run()
-	return console.as_dict()
\ No newline at end of file
+	return console.as_dict()
+
+@frappe.whitelist()
+def show_processlist():
+	frappe.only_for('System Manager')
+	return frappe.db.sql('show processlist', as_dict=1)
```

Comment:
Postgres support for this?

Reply:
Let's maybe use "SHOW FULL PROCESSLIST" to see whole query.

Reply:
@surajshetty3416 those who want postgres support will add it 😜

@sagarvora sure, let me do that

Reply:
Kill process?

We can sponsor this feature, if required.